lib/fog/core/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fog
   module Core
-    VERSION = "1.42.0"
+    VERSION = "1.42.1"
```

Comment:
Could you remove this change please?
Is better to leave the bump to the moment of the release.